ios/Podfile

```diff
@@ -1,14 +1,51 @@
-# Uncomment the next line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
```

Comment:
Is this intended?

Reply:
I used to follow the rn template, as the next version will come to drop ios 9 and less https://github.com/facebook/react-native/releases/tag/v0.63.0-rc.1 and I expect changes here